LICENSE-binary

```diff
@@ -205,172 +205,69 @@ This project bundles some components that are also licensed under the Apache
 License Version 2.0:
 
 com.101tec:zkclient:0.7
-com.amazonaws:aws-java-sdk:1.7.4
```

Comment:
why we removed so many dependencies?
E.g. this aws-java-sdk is used in pinot-s3 plugin

Reply:
I ran the command as specified in the release process:https://cwiki.apache.org/confluence/display/PINOT/Creating+an+Apache+Release#CreatinganApacheRelease-UpdatingLICENSE-binaryfile

The result does not include the plugins.

Reply:
hmm, i see. 
@mcvsubbu @snleee do you know if we should include the plugins dependencies here as well?

Reply:
I would think so, since we are including the plugins along with the product. But I am not sure. Also not sure how to get the plugins into the dependencies or why it does not show up

Reply:
I think pinot-distribution module just copied the jars from pinot-plugins.
@chenboat can you try to explicitly add pinot-plugins module into pinot-distributions's pom as a dependency then run the command to see if plugins dependencies are added

Reply:
Thanks @fx19880617. Adding the pinot plugins modules into pinot-distribution's pom (and also the top level pom) can work. But it looks like to me that the pinot-distribution's pom only takes a small set of core modules to avoid bringing too many external dependencies. This outdated doc explains the organization: 
https://github.com/apache/incubator-pinot/blob/master/docs/code_modules.rst

I wonder if we just want to add all the plugin modules to pinot distribution's pom to derive the set of licenses.

Add @kishoreg here as well how what module we should add to Pinot distributions.

Reply:
I added the s3 plugin but there is still no com.amazonaws:aws-java-sdk:1.7.4 dep. Could aws change their dep structure?